docs/components/store.md

```diff
@@ -274,30 +274,44 @@ While the remaining settings are **optional**:
 
 ## Caching Bucket
 
-Thanos Store Gateway supports a "caching bucket" with chunks caching to speed up loading of chunks from TSDB blocks. Currently only memcached "backend" is supported:
+Thanos Store Gateway supports a "caching bucket" with chunks and metadata caching to speed up loading of chunks from TSDB blocks. To configure caching, one needs to use `--store.caching-bucket.config=<yaml content>` or `--store.caching-bucket.config-file=<file.yaml>`.
+
+Currently only memcached "backend" is supported:
 
 ```yaml
 backend: memcached
 backend_config:
   addresses:
     - localhost:11211
 
-caching_config:
-  chunk_subrange_size: 16000
-  max_chunks_get_range_requests: 3
-  chunk_object_size_ttl: 24h
-  chunk_subrange_ttl: 24h
+chunk_subrange_size: 16000
+max_chunks_get_range_requests: 3
+chunk_object_size_ttl: 24h
+chunk_subrange_ttl: 24h
+blocks_iter_ttl: 5m
+metafile_exists_ttl: 2h
+metafile_doesnt_exist_ttl: 15m
+metafile_content_ttl: 24h
+metafile_max_size: 1MiB
 ```
 
 `backend_config` field for memcached supports all the same configuration as memcached for [index cache](#memcached-index-cache).
 
-`caching_config` is a configuration for chunks cache and supports the following optional settings:
+`caching_config` is a configuration for cache and supports the following optional settings for chunks caching:
```

Comment:
But `caching_config` key doesn't exist, right? :thinking: So, I think this line should be reworded.

Reply:
You're correct. I've removed mention of `caching_config`.